src/AccessibilityInsights.SharedUx/Telemetry/Logger.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AccessibilityInsights.SharedUx.Telemetry
+{
+    /// <summary>
+    /// Use this class to send telemetry from anywhere within
+    /// the Accesseibility Insights code. This does not include extensions,
```

Comment:
small typo, extra 'e' in Accessibility. auto-commitable suggestion:
```suggestion
    /// the Accessibility Insights code. This does not include extensions,
```